constraint/pkg/client/drivers/interface.go

```diff
@@ -5,35 +5,43 @@ import (
 
 	"github.com/open-policy-agent/frameworks/constraint/pkg/core/templates"
 	"github.com/open-policy-agent/frameworks/constraint/pkg/types"
+	"github.com/open-policy-agent/opa/storage"
 	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
 )
 
-type QueryCfg struct {
-	TracingEnabled bool
+// A Driver implements Rego query execution of Templates and Constraints.
+type Driver interface {
+	AddTemplate(ct *templates.ConstraintTemplate) error
```

Comment:
nit: should we document what these interface functions do?

Reply:
Done.